core/templates/domain/state_card/state-card.model.spec.ts

```diff
@@ -125,7 +125,8 @@ describe('State card object factory', () => {
       'content', audioTranslationLanguageService);
     _sampleCard2 = StateCard.createNewCard(
       'State 2', '<p>Content</p>', '',
-      null, null, writtenTranslationsObjectFactory.createEmpty(),
+      {} as Interaction, {} as RecordedVoiceovers,
```

Comment:
Can we not use this explicit typecasting

Reply:
test file